.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -25,7 +25,7 @@ Welcome to the PyInstaller issue tracker! Before creating an issue, please heed
 ## Context information (for bug reports)
 
 * Output of `pyinstaller --version`: ```(paste here)```
-* Version of Python: e.g. 3.7
+* Version of Python: e.g. 3.8
```

Comment:
There is no point changing this; 3.7 is still a valid version. Please remove this change.

Reply:
The point was just to have the most recent version defaulted into the template, as that's where most bugs would surface.